x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -340,7 +340,7 @@ export class FleetPlugin
                         includeIn: 'all',
                       },
                       {
-                        id: `${PLUGIN_ID}-agents-read`,
+                        id: `agents_read`,
```

Comment:
I updated those to have the same naming convention as other plugins for subfeature (it's a newly introduced privilege behind a feature flag so it should be okay to change)